pallets/subtensor/tests/block_step.rs

```diff
@@ -5,6 +5,7 @@ use mock::*;
 use sp_core::U256;
 
 #[test]
+#[allow(clippy::unwrap_used)]
```

Comment:
Does it make sense to allow unwraps for all the tests ?

Reply:
yes, tests don't affect the runtime, so it can use unwraps or direct indexing.